app/src/main/res/layout/bottom_right_overlay.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+    <variable
+      name="listener"
+      type="org.oppia.android.app.spotlight.SpotlightNavigationListener" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:id="@+id/spotlight_overlay_constraint_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/transparent_background"
+    android:clickable="true"
+    android:focusable="true">
+
+    <ImageView
+      android:id="@+id/spotlight_overlay_arrow"
+      android:layout_width="@dimen/arrow_width"
+      android:layout_height="@dimen/arrow_height"
+      android:rotationX="180"
+      android:rotationY="180"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      app:srcCompat="@drawable/ic_rounded_arrow_up_right" />
+
+    <TextView
+      android:id="@+id/custom_text"
+      style="@style/TextViewStart"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="10dp"
+      android:background="@drawable/spotlight_overlay_hint_background"
+      android:paddingStart="8dp"
+      android:paddingTop="4dp"
+      android:paddingEnd="4dp"
+      android:paddingBottom="4dp"
+      android:textSize="20dp"
+      app:layout_constraintBottom_toTopOf="@id/spotlight_overlay_arrow"
+      app:layout_constraintEnd_toEndOf="@+id/spotlight_overlay_arrow"
+      app:layout_constraintStart_toStartOf="parent" />
+
+    <TextView
+      android:id="@+id/close_target"
+      style="@style/StateButtonActive"
+      android:layout_width="160dp"
+      android:layout_height="48dp"
+      android:layout_marginTop="50dp"
```

Comment:
Use mutliple of `4` maybe `48dp` or `52dp`

Reply:
done